lib/handlers/patch.js

```diff
@@ -18,8 +18,17 @@ const PATCH_PARSERS = {
   'text/n3': require('./patch/n3-patch-parser.js')
 }
 
+// use mime-type as contentType for new rdf resource
```

Comment:
```suggestion
// use media-type as contentType for new RDF resource
```
MIME types became media types some years ago. We should use current terminology whenever possible.